weave

```diff
@@ -1021,32 +1057,37 @@ case "$COMMAND" in
         ;;
     start)
         collect_cidr_args "$@"
-        shift $CIDR_COUNT
+        shift $CIDR_ARG_COUNT
         [ $# -eq 1 ] || usage
-        CONTAINER=$(docker start $1)
+        docker start $1 > /dev/null
```

Comment:
I'm unclear which part you think is better.  Returning the ID passed in seems worse, to me.